test/unit/test_devices/test_iosxewlc_device.py

```diff
@@ -0,0 +1,135 @@
+import mock
+import pytest
+
+from pyntc.devices import IOSXEWLCDevice
+from pyntc.devices import iosxewlc_device as iosxewlc_module
+
+
+def test_show(iosxewlc_send_command):
+    command = "show_ip_arp"
+    device = iosxewlc_send_command([f"{command}.txt"])
+    device.show(command)
+    device.native.send_command.assert_called_with(command_string="show_ip_arp")
+    device.native.send_command.assert_called_once()
+
+
+def test_show_delay_factor(iosxewlc_send_command):
+    command = "show_ip_arp"
+    delay_factor = 20
+    device = iosxewlc_send_command([f"{command}"])
+    device.show(command, delay_factor=delay_factor)
+    device.native.send_command.assert_called_with(command_string="show_ip_arp", delay_factor=delay_factor)
+    device.native.send_command.assert_called_once()
+
+
+# Test install mode upgrade
+@mock.patch.object(IOSXEWLCDevice, "_image_booted")
+@mock.patch.object(IOSXEWLCDevice, "set_boot_options")
+@mock.patch.object(IOSXEWLCDevice, "show")
+@mock.patch.object(IOSXEWLCDevice, "_wait_for_device_reboot")
+@mock.patch.object(IOSXEWLCDevice, "_wait_for_device_start_reboot")
+@mock.patch.object(IOSXEWLCDevice, "_get_file_system")
+def test_install_os_install_mode(
```

Comment:
`IOSXEWLCDevice.fast_cli` should be mocked and asserted that it was called as expected

Reply:
This has been updated.